zyte_common_items/log_formatters.py

```diff
@@ -0,0 +1,17 @@
+from logging import INFO
+
+from scrapy.exceptions import DropItem
+from scrapy.logformatter import LogFormatter
+
+
+class InfoDropItem(DropItem):
+    """DropItem subclass for items that should be dropped with an INFO message
+    (instead of the default WARNING message)."""
+
+
+class ZyteLogFormatter(LogFormatter):
```

Comment:
Alternatively, we could allow to configure the severity level at Scrapy level via settings given that this is not the first time this has come up.

Reply:
@kmike mentioned somewhere that maybe we should allow components to set their desired log level in the exception, e.g. `raise DropItem(severity=INFO)`.

Maybe we could have a setting to implement the overall default log level, allow components to set a custom value through severity, and let components choose whether or not they respect the setting, force a given value, or provide their own setting to override their own request droppings.

Reply:
https://github.com/scrapy/scrapy/issues/6603